src/thx/Arrays.hx

```diff
@@ -1160,6 +1161,17 @@ Finds the min element of the array given the specified ordering.
 
     return Options.toFailure(Nel.fromArray(collisions), m);
   }
+
+  public static function toStringMap<V>(arr: ReadonlyArray<Tuple<String, V>>): StringMap<V> {
```

Comment:
In any case, if it does then the caller can use is as it is... this follows the principle of always accept the most general type possible, and return the most specific.